### PR TITLE
chore: remove add local user from UI 

### PR DESCRIPTION
**Pull request by nguyen102**
Issue #, if available:

Description of changes:
* Remove update internal auth user api
* Remove `Add Local User` UI component
* Fix `AddSingleUser.test.js` test. This test was calling the wrong UI component

There's two versions of this code, the [first one](https://github.com/awslabs/service-workbench-on-aws/blob/feat-internal-auth-dep/addons/addon-base-rest-api/packages/base-controllers/lib/users-controller.js#L67) was removed, but we need to remove the second one as well. 

Checklist:

- [x] Have you successfully deployed to an AWS account with your changes?
- [x] Have you written new tests for your core changes, as applicable?

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: addons/addon-base-raas/packages/base-raas-rest-api/lib/controllers/users-controller.js [hunk 1]
```diff
@@ -74,22 +74,23 @@ async function configure(context) {
     }),
   );
 
+  // TODO: Do we want to support changing user passwords? Currently this API is only supported for internal Auth
```
Comment (SanketD92) [inline addons/addon-base-raas/packages/base-raas-rest-api/lib/controllers/users-controller.js]: I think there's no need to support password change, if login API is gone anyways.

File: addons/addon-base-raas/packages/base-raas-rest-api/lib/controllers/users-controller.js [hunk 2]
```diff
@@ -74,22 +74,23 @@ async function configure(context) {
...
-      const requestContext = res.locals.requestContext;
-      const username = req.params.username;
-      const { password } = req.body;
-
-      // Save password salted hash for the user in internal auth provider (i.e., in passwords table)
-      await dbPasswordService.savePassword(requestContext, { username, password });
-      res.status(200).json({ username, message: `Password successfully updated for user ${username}` });
-    }),
-  );
+  // router.put(
```
Comment (SanketD92) [inline addons/addon-base-raas/packages/base-raas-rest-api/lib/controllers/users-controller.js]: Could we also update API changes in the openapi file?

File: addons/addon-base-raas/packages/base-raas-rest-api/lib/controllers/users-controller.js (hunk 1 above)
Comment (nguyen102) [reply addons/addon-base-raas/packages/base-raas-rest-api/lib/controllers/users-controller.js]: deleted

File: addons/addon-base-raas/packages/base-raas-rest-api/lib/controllers/users-controller.js (hunk 2 above)
Comment (nguyen102) [reply addons/addon-base-raas/packages/base-raas-rest-api/lib/controllers/users-controller.js]: good idea. Updated OpenApi file.

**Review by SanketD92 — APPROVED** [review]
LGTM

**Pull request merged**